QUEST_LV_0100.tsv

```diff
@@ -4198,176 +4198,176 @@ QUEST_LV_0100_20150317_004197	$Destroy the West Barrier in Nuotori Cliff
+QUEST_LV_0100_20150317_004326	$Search District 6 for Purifier Parts
+QUEST_LV_0100_20150317_004327	$The Mine Compass is pointing towards District 6. Follow it and find parts to repair the purifier.
 QUEST_LV_0100_20150317_004328	 Retrieve %s from Specter Monarch
-QUEST_LV_0100_20150317_004329	Repair the Passage Purifier on the 1st Floor
-QUEST_LV_0100_20150317_004330	Found the parts for repairing Passage Purifier. Assemble and activate the purifier.
-QUEST_LV_0100_20150317_004331	Check circulation purifier system
-QUEST_LV_0100_20150317_004332	Circulation Purifier of 2nd GAllery is not working. Find out what the problem is.
-QUEST_LV_0100_20150317_004333	Repair the Purifier Pipe
-QUEST_LV_0100_20150317_004334	Circulation purifier is unable to absorb {nl}the toxic fumes. Check the purifier in district 3 and fix it.
-QUEST_LV_0100_20150317_004335	Check circulation purifier
-QUEST_LV_0100_20150317_004336	Purifier in District 3 seems to work properly. Activate the circulation purifer again.
-QUEST_LV_0100_20150317_004337	Check Purifier Pipe in District 2
-QUEST_LV_0100_20150317_004338	Check if the purifier is 2nd Gallery is working properly.
-QUEST_LV_0100_20150317_004339	Defeat the Carapace
-QUEST_LV_0100_20150317_004340	Activate Circulation Purifier
-QUEST_LV_0100_20150317_004341	All Purifier Pipes seems to work properly. Activate the circulation purifier.
-QUEST_LV_0100_20150317_004342	Check the secondary purifier in 2nd Gllery
-QUEST_LV_0100_20150317_004343	Secondary Purifier in 2nd Gallery has stopped working. Find out why it stopped.
-QUEST_LV_0100_20150317_004344	Use Mining Compass
-QUEST_LV_0100_20150317_004345	Use mining compass to check what you need to fix the secondary purifier.
-QUEST_LV_0100_20150317_004346	Check the spell device
-QUEST_LV_0100_20150317_004347	 The mine compass is pointing to the spell device in 4th area. Check what the problem is. 
-QUEST_LV_0100_20150317_004348	Use mining compass to check what you need to fix the secondary purifier.
-QUEST_LV_0100_20150317_004349	Search for tools to remove rust
-QUEST_LV_0100_20150317_004350	Follow the compass to find a way to remove the rust.
-QUEST_LV_0100_20150317_004351	Restore the spell device in district 4
-QUEST_LV_0100_20150317_004352	Seems like you can fix the spell device in Area 4 with lubricant.
-QUEST_LV_0100_20150317_004353	Activate secondary purifier in 2nd Gallery
-QUEST_LV_0100_20150317_004354	Seem like spell supply is working well. Activate the secondary purifier.
-QUEST_LV_0100_20150317_004355	Check the main purifier
-QUEST_LV_0100_20150317_004356	Main purifier is not working. Go and check on it.
-QUEST_LV_0100_20150317_004357	Use Mine Compass to locate the lost parts
+QUEST_LV_0100_20150317_004329	$Repair the Passage Purifier on the 1st Floor
+QUEST_LV_0100_20150317_004330	$Found the parts needed to repair the Passage Purifier. Try assembling and activating the purifier.
```

Comment:
Try assembling and activating the purifier. -> Try to assemble and activate it (*shorter*)